pkg/scheduler/framework/interface.go

```diff
@@ -250,10 +250,10 @@ func (s *Status) IsSkip() bool {
 	return s.Code() == Skip
 }
 
-// IsUnschedulable returns true if "Status" is Unschedulable (Unschedulable or UnschedulableAndUnresolvable).
-func (s *Status) IsUnschedulable() bool {
+// IsAbortStatus returns true if "Status" is Unschedulable (Unschedulable, UnschedulableAndUnresolvable, or Pending).
+func (s *Status) IsAbortStatus() bool {
```

Comment:
Just my personal opinion, the old name (`IsUnschedulable`) seems easier to understand.

Reply:
We have to rename like this if we include Pending because Pending isn't unschedulable.
In other words, we can keep `IsUnschedulable` if we create another function like `IsPending`, but then we need to `s.IsUnschedulable() && s.IsPending()` all the places instead of `s.IsAbortStatus()`. 
I still prefer to rename. (but maybe we have better name on it?)

Reply:
What about `IsRejected`?
No need to add `Status` because this is a method of the `Status` struct already.